code/script.js

```diff
@@ -1,47 +1,160 @@
-// All the DOM selectors stored as short variables
+// Declare variables
+const startButton = document.getElementById('start-button');
+const startContent = document.getElementById('start-content');
+const chatBox = document.getElementById('chat-box');
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper');
 
-// Global variables, if you need any, declared here
+let buttonText = "";
+let questionCounter = 0;
 
-// Functions declared here
+// Start chat by moving past start page
+startButton.addEventListener('click', () => {
+  startContent .style.display = "none";
+  chatBox.style.display = "flex";
+  questionGenerator();
+});
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Function to move through questions
+const questionGenerator = () => {
+  questionCounter++;
+  if (questionCounter === 1) {
+    setTimeout(question1, 1000);
+  } else if (questionCounter === 2) {
+    setTimeout(question2, 1000, nameInput.value);
+  } else if (questionCounter === 3) {
+    setTimeout(question3, 1000, buttonText);
```

Comment:
So buttonText is the same parameter that's passed as parameter in line 63 but called answer there instead, in which buttonText is a variable described on line 132 but declared on line 10 as a global variable. What if we move line 132 out as a global variable, change line 131 to ```showMessage('buttonText', 'user')```, remove buttonText parameter in line 28, remove answer parameter in line 63, and changes line 65 to ``` if (buttonText ==='Great!')```. Would that work?